src/app/components/grid/client.ts

```diff
@@ -0,0 +1,90 @@
+import { Socket, Channel } from 'phoenix';
+import { Growls } from '../../../lib/gj-lib-client/components/growls/growls.service';
+import { Environment } from '../../../lib/gj-lib-client/components/environment/environment.service';
+import { store } from '../../store/index';
+import { getCookie } from '../../../_common/cookie/cookie.service';
+import {
+	Notification,
+	getNotificationText,
+} from '../../../lib/gj-lib-client/components/notification/notification-model';
+
+export class GridClient {
+	connected = false;
+	socket: Socket | null = null;
+	channels: Channel[] = [];
+
+	constructor() {
+		this._init();
+	}
+
+	private async _init() {
+		if (GJ_IS_SSR) {
+			return;
+		}
+
+		const cookie = await getCookie('frontend');
+		const user = store.state.app.user;
+
+		if (user === null || cookie === undefined) {
+			// not properly logged in
+			return;
+		}
+
+		console.log('Connecting to the Grid!');
+
+		const userId = user.id.toString();
+
+		this.socket = new Socket(Environment.gridHost);
+		this.socket.connect({
```

Comment:
got this message: `passing params to connect is deprecated. Instead pass :params to the Socket constructor`